demonstrations/tutorial_quantum_volume.py

```diff
@@ -0,0 +1,732 @@
+r""".. _quantum_volume:
+
+Quantum volume
+==============
+
+.. meta::
+    :property="og:description": Learn about quantum volume, and how to 
+        compute it.
+    :property="og:image": https://pennylane.ai/qml/_images/bloch.png
+
+Twice per year, a project called the TOP500 [#top500]_ releases a list of the
+500 most powerful supercomputing systems in the world. However, there is a large
+amount of variation in how supercomputers are built. `Some
+<https://en.wikipedia.org/wiki/Fugaku_(supercomputer)>`_ use 48-core processors,
+while `others <https://en.wikipedia.org/wiki/Sunway_TaihuLight>`_ use processors
+with up to 260 cores. The speed of processors will differ, and they may be
+connected in different ways. We can't rank them by simply counting the number of
+processors. In order to make a fair comparison, we need benchmarking standards
+that give us a holistic view of their performance. To that end, the TOP500
```

Comment:
```suggestion
that give us a holistic view of their performance.

To that end, the TOP500
```

Suggest making this a new paragraph (since it's quite a big block of text when rendered!).

Starting sentence of the new paragraph might need some modification though.

Reply:
I've gone back and forth with this a lot with the intention of trying to make the intro more compact. However I agree it should be separated into more paragraphs. While the cost is that it looks slightly longer, I think it also makes it more digestible.

Reply:
I find more and shorter paragraphs generally better for web content (news articles -- sometimes paragraphs can be 1-2 sentences long! -- blogs, tutorials) and longer paragraphs better suited to 'heftier' mediums (papers, manuscripts, books)